edgelet/docker-rs/src/models/container_create_body.rs

```diff
@@ -16,20 +16,24 @@ use serde_json::Value;
 // We do not want to restrict the properties that the user can set in their create options, because future versions of Docker can add new properties
 // that we don't define here.
 //
-// So this type has a `#[serde(flatten)] HashMap` field to collect all the extra properties that we don't have a struct field for.
+// So this type has a `#[serde(flatten)] BTreeMap` field to collect all the extra properties that we don't have a struct field for.
 //
 // But if an existing field references another type under `crate::models::`, then that would still be parsed lossily, so we would have to also add
-// a `#[serde(flatten)] HashMap` field there. And if that type has fields that reference types under `crate::models::` ...
+// a `#[serde(flatten)] BTreeMap` field there. And if that type has fields that reference types under `crate::models::` ...
 //
 // To avoid having to do this for effectively the whole crate, instead we've just commented out the fields we don't use in our code.
 //
+// Note: We're using BTreeMap instead of HashMap because iotedged stores an HMAC of its local config (whose object representation uses this struct)
```

Comment:
`s/an HMAC of/a hash of/g`

HMAC takes a hash and signs it with a key. We just do a hash.

Reply:
Oops, thanks

Reply:
Fixed everywhere (including PR title/description).